### PR TITLE
1.12.5

### PR DESCRIPTION
**Pull request by GeoffreyBooth**
Per the changelog:

*   Better handling of `default`, `from`, `as` and `*` within `import` and `export` statements. You can now import or export a member named `default` and the compiler won’t interpret it as the `default` keyword. #4492, #4483
*   Fixed a bug where invalid octal escape sequences weren’t throwing errors in the compiler. #4490

I also updated the docs generation using code from #4494, to use Markdown-It instead of Marked.

### CODE REVIEW

File: documentation/v1/docs.css [hunk 1]
```diff
@@ -44,7 +44,6 @@ li {
 }
 table {
   margin: 16px 0 0 13px; padding: 0;
-  width: 690px;
```
Comment (lydell) [inline documentation/v1/docs.css]: Is this change intended? Won't it make table much wider on large screens?

Comment (GeoffreyBooth) [inline documentation/v1/docs.css]: This was stretching out the table of operators, which is why I took it out. The table of usage flags is now just the width of the code examples: http://rawgit.com/geoffreybooth/coffeescript/1.12.5/docs/v1/index.html

Comment (lydell) [inline documentation/v1/docs.css]: Nice! 👍

**Comment by GeoffreyBooth** [comment]
@lydell if this looks good to you, do you mind cutting the release? 😄

**Comment by lydell** [comment]
I'll try to remember doing it tonight.

**Comment by lydell** [comment]
@GeoffreyBooth Are you publishing to npm and stuff? Otherwise I'll go ahead and do that now.

**Comment by GeoffreyBooth** [comment]
I’m happy for you to do it 😄 I think when I published to `coffeescript` (no hyphen) I took the hyphen out of `package.json` before publishing. I wish I had written down exactly what I did. Maybe if you `npm install coffeescript@1.12.4` and compare it against `npm install coffee-script@1.12.4` you can see, or I can look into it but I won’t have time until Wednesday.
